crud/crud.cpp

```diff
@@ -83,8 +74,15 @@ crud::set_leader_info(bzn::message& msg)
 }
 
 
+bool validate_create(const bzn::message /*&msg*/)
+{
+    return false;
+}
+
+
+
 void
-crud::handle_create(const bzn::message& msg)
+crud::commit_create(const bzn::message &msg)
 {
     storage_base::result result = this->storage->create(msg["db-uuid"].asString(), msg["data"]["key"].asString(), msg["data"]["value"].asString());
```

Comment:
auto result = ...

Reply:
Went through crud.cpp and crud_test.cpp and changed the local variable declarations to auto.